src/blueapi/startup/example.py

```diff
@@ -1,38 +1,74 @@
 from ophyd.sim import Syn2DGauss, SynGauss, SynSignal
 
-from blueapi.plans import *  # noqa: F401, F403
-
 from .simmotor import BrokenSynAxis, SynAxisWithMotionEvents
 
-x = SynAxisWithMotionEvents(name="x", delay=1.0, events_per_move=8)
-y = SynAxisWithMotionEvents(name="y", delay=3.0, events_per_move=24)
-z = SynAxisWithMotionEvents(name="z", delay=2.0, events_per_move=16)
-theta = SynAxisWithMotionEvents(
-    name="theta", delay=0.2, events_per_move=12, egu="degrees"
-)
-x_err = BrokenSynAxis(name="x_err", timeout=1.0)
-sample_pressure = SynAxisWithMotionEvents(
-    name="sample_pressure", delay=30.0, events_per_move=128, egu="MPa", value=0.101
-)
-sample_temperature = SynSignal(
-    func=lambda: ((x.position + y.position + z.position) / 1000.0) + 20.0,
+
+def x(name="x") -> SynAxisWithMotionEvents:
```

Comment:
Is the `name` field always required and is it always going to be the same as the function name?

Reply:
No it is not required as an argument, as long as the device return has one. I just copied the dodal format. It also doesn't matter what the function is called. The function name is used to determine if others depend on it and instantiate the devices in order, but the resulting dictionary is `device.name: device`.

Reply:
It's not always going to be the same as the function name: https://nsls-ii.github.io/ophyd/architecture.html#names